go.sum

```diff
@@ -86,6 +86,14 @@ github.com/DATA-DOG/go-sqlmock v1.3.3/go.mod h1:f/Ixk793poVmq4qj/V1dPUg2JEAKC73Q
 github.com/DATA-DOG/go-sqlmock v1.4.1/go.mod h1:f/Ixk793poVmq4qj/V1dPUg2JEAKC73Q5eFN3EC/SaM=
 github.com/DataDog/agent-payload v4.55.0+incompatible h1:jnpxs+jmh9DY2wFIlF4meC7XB7yAVtV5e3+YbzJ69aM=
 github.com/DataDog/agent-payload v4.55.0+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.57.1-0.20210226201456-78fc04b9f1a0+incompatible h1:YxqLYpmamxsTLyMXj0T3RamlDtKInA7BavFgPPoWVyY=
+github.com/DataDog/agent-payload v4.57.1-0.20210226201456-78fc04b9f1a0+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.57.1-0.20210226204340-7aeb74ed128c+incompatible h1:SjM1em1UPLyBAKFSSx9h7zoud5qMTTHo9XPKqRA9wUw=
+github.com/DataDog/agent-payload v4.57.1-0.20210226204340-7aeb74ed128c+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.57.1-0.20210226213815-d5f705b5a0d0+incompatible h1:W3EzS0FcIklcpH7TSHGYwf0V1Xs5Dnr8Lpe4GPl+bd4=
+github.com/DataDog/agent-payload v4.57.1-0.20210226213815-d5f705b5a0d0+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.57.1-0.20210304080724-ce2d654862ac+incompatible h1:KBQh4rG0bI+ZenXCNryChQ8FdJxUQAsqh0Io64mqEFk=
+github.com/DataDog/agent-payload v4.57.1-0.20210304080724-ce2d654862ac+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
```

Comment:
You can run `go mod tidy` to cleanup all these extra versions.

Reply:
Will do! I'm just waiting for a ✅  here before merging & releasing the agent-payload side, after which I'll update this dependency & clean it up